src/confdb/confdb.c

```diff
@@ -1570,6 +1590,24 @@ static errno_t confdb_init_domain_subdomains(struct sss_domain_info *domain,
             CONFDB_DOMAIN_SUBDOMAIN_REFRESH_DEFAULT_VALUE;
     }
 
+    ret = get_entry_as_uint32(res->msgs[0],
+                              &domain->subdomain_refresh_interval_offset,
+                              CONFDB_DOMAIN_SUBDOMAIN_REFRESH_OFFSET,
+                              CONFDB_DOMAIN_SUBDOMAIN_REFRESH_OFFSET_DEFAULT_VALUE);
+    if (ret != EOK) {
+        DEBUG(SSSDBG_FATAL_FAILURE,
+              "Invalid value for [%s]\n", CONFDB_DOMAIN_SUBDOMAIN_REFRESH_OFFSET);
+        goto done;
+    } else if (domain->subdomain_refresh_interval_offset == 0) {
```

Comment:
Hi,

why should the offset not be `0`? For `subdomain_refresh_interval` we do not want it to be `0` because there should be a refresh. But I think it would be ok if the offset is `0`.

bye,
Sumit

Reply:
Hi Sumit,

You are right. I will remove this check.